_locales/cs/maqueenPlusV2-jsdoc-strings.json

```diff
@@ -1,23 +1,52 @@
 {
     "maqueenPlusV2.controlMotor": "Blok pro ovládání motorů. Umožňuje měnit směr a rychlost otáček.",
-    "maqueenPlusV2.controlLED": "Blok pro ovládání předních LED světel.",
```

Comment:
I would make this a bit more conscious: `"Umožňuje měnit směr a rychlost otáčení motorů.".
The same for the `maqueenPlusV2.controlLED`.

Reply:
OK, why not.